extras/userdoc/md/documentation/installation.md

```diff
@@ -71,6 +79,281 @@ search paths. For example, if you are using bash:
+
+5.  Extract the NEST tarball as a subdirectory in that directory or clone NEST from Github into a subdirectory. 
+
+        mkdir NEST       # directory for all NEST stuff
+        cd NEST
+        tar zxf  nest-2.12.0.tar.gz
+        mkdir bld
+        cd bld
+
+6.  Configure and build NEST inside that directory:
+
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=</install/path> \
+          -DCMAKE_C_COMPILER=gcc-6\
+          -DCMAKE_CXX_COMPILER=g++-6 \
+          </path/to/NEST/src>
+
+    make -j4         # -j4 builds in parallel using 4 processes
+    make install
+    make installcheck
+
+To compile NEST with MPI support, add `-Dwith-mpi=ON` as `cmake` option.
+
+### Python on Mac
+
+The version of Python shipping with OSX/macOS is rather dated and does not
+include key packages such as NumPy. Therefore, you need to install Python via
+a channel that provides scientific packages.
+
+One well-tested source is the [Anaconda](https://www.continuum.io/anaconda-overview) 
+Python distribution for both Python 2 and 3. If you do not want to install the
+full Anaconda distribution, you can also install [Miniconda](http://conda.pydata.org/miniconda.html)
+and then install the packages needed by NEST by running
+
+    conda install numpy scipy matplotlib ipython cython nose
+
+Alternatively, you should be able to install the necessary Python packages via Homebrew, but
+this has not been tested.
+
+## Troubleshooting
+
+If your operating system does not find the `nest` executable or Python does not
+find the `nest` module, your path variables may not be set correctly. This may
+also be the case if Python cannot load the `nest` module due to missing or
+incompatible libraries. In this case, please run
+
+    source </path/to/nest_install_dir>/bin/nest_vars.sh
+
+to set the necessary environment variables. You may want to include this line
+in your `.bashrc` file, so that the environment variables are set automatically.
+
+## Windows
+
+Windows and Linux differ considerably. This is the reason why it is difficult
+to compile NEST natively under Windows. However, it is still possible to use
+NEST under Windows using one of the following methods:
+
+### Using the NEST LiveCD
+
+This is the easiest way to use NEST without having to install anything on your
+computer. The LiveCD can be used in almost any computer and boots directly into
+a complete Ubuntu system, which already has NEST, Python, and some analysis
+tools installed.
+The CD image is available on the [download page](download.md).
+
+### Virtual Machines
+
+A virtual machine is a software that lets you use Linux in parallel to Windows.
+We recommend to use VirtualBox, which is free software and can be downloaded
+from <http://www.virtualbox.org/>. Another popular virtual machine software is <http://www.vmware.com/>.
+The easiest way to get started is to download the NEST LiveCD from the [download page](download.md)
+and either just boot the CD image directly or install it into a virtual hard
+disk.
```

Comment:
This information is outdated. We now have proper OVA images that can be directly imported by VM software.